GiftModels/FinancialDetails.cs

```diff
@@ -18,7 +18,26 @@ public class FinancialDetails
 
         public virtual decimal TotalPremiumAmount { get; set; }
 
-        public virtual string Prem { get; set; } // Y/N
+        public virtual string Prem

```

Comment:
First of all, now that i think about it, why not just show them Y/N?

Otherwise try this:  `return string.Compare(prem, "y", StringComparison.OrdinalIgnoreCase) ? "Yes" : "No";`